src/fury/thirdparty/MurmurHash3.cc

```diff
@@ -14,14 +14,22 @@
 
 // Microsoft Visual Studio
 
+inline uint32_t rotl32(uint32_t x, int8_t r) {
+  return (x << r) | (x >> (32 - r));
+}
+
+inline uint64_t rotl64(uint64_t x, int8_t r) {
+  return (x << r) | (x >> (64 - r));
+}
+
 #if defined(_MSC_VER)
 
 #define FORCE_INLINE __forceinline
 
 #include <cstdint>
 
-#define ROTL32(x, y) _rotl(x, y)
-#define ROTL64(x, y) _rotl64(x, y)
+#define ROTL32(x, y) rotl32(x, y)
+#define ROTL64(x, y) rotl64(x, y)
```

Comment:
Could you explain this change? I don't think it need to be changed.

https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/rotl-rotl64-rotr-rotr64?view=msvc-170

Reply:
Sorry, a missing header file caused a compilation error. I will modify it back.